docs/admin/templates/extending-templates/parameters.md

```diff
@@ -313,6 +313,60 @@ data "coder_parameter" "project_id" {
 }
 ```
 
+## Workspace presets
+
+Workspace presets allow you to configure commonly used combinations of parameters
+into a single option, which makes it easier for developers to pick one that fits
+their needs.
+
+Use `coder_workspace_preset` to define the preset parameters.
+After you save the template file, the presets will be available for all new
+workspace deployments.
```

Comment:
@bpmct @dannykopping we _could_ be more explicit here if you think it needs it, but I think this is probably concise enough

Reply:
Nope this seems perfect 👍